src/app/pages/send/send-crypto-asset-form/form/btc/btc-send-form.tsx

```diff
@@ -90,7 +93,23 @@ export function BtcSendForm() {
                   symbol={symbol}
                 />
                 <BitcoinRecipientField />
-                {currentNetwork.chain.bitcoin.bitcoinNetwork === 'testnet' && <TestnetBtcMessage />}
+
+                {/* TESTING */}
+                <Callout variant="warning">
+                  <CalloutLayout
+                    caption={
+                      <>
+                        {'This is a Bitcoin testnet transaction. Funds have no value. '}
+                        <Link href="https://coinfaucet.eu/en/btc-testnet" textStyle="caption.02">
+                          Get testnet BTC here ↗
+                        </Link>
+                      </>
+                    }
+                    variant="warning"
```

Comment:
I don't love setting the `variant` here and on the `Callout` so might rethink this.